src/dashboard/src/components/helpers.py

```diff
@@ -219,20 +219,17 @@ def get_atom_levels_of_description(clear=True):
     :param bool clear: When True, deletes all existing levels of description from the Archivematica database before fetching; otherwise, the fetched levels of description will be appended to the already-stored values.
     :raises AtomError: if no AtoM URL or authentication credentials are defined in the settings, or if the levels of description cannot be fetched for another reason
     """
-    url = get_setting('dip_upload_atom_url')
-    if not url:
+    url = get_setting('dip_upload_atom_url', None)
+    if url is None:
```

Comment:
Is an empty string a valid AtoM URL?  If not, leaving it as `if not url` may be better.